plugins/woocommerce-admin/client/homescreen/mobile-app-modal/components/SendMagicLinkButton.tsx

```diff
@@ -9,7 +9,12 @@ export const SendMagicLinkButton = ( {
 }: {
 	onClickHandler: () => void;
 } ) => (
-	<Button className="send-magic-link-button" onClick={ onClickHandler }>
+	<Button
+		className="send-magic-link-button"
+		onClick={ () => {
+			onClickHandler();
```

Comment:
I'm curious why this change is required?

Reply:
argh I didn't revert correctly - I mistakenly put the install-and-connect tracks in here previously and noticed it at some point, so I removed the recordEvent only